### DISCUSSION TITLE
Использование serviceDocxGenerator.

### DISCUSSION THREAD

**Discussion by charados** [totumonline/totum-ru-issues-and-discussions#615] [Как это сделать?] [answered]
Не подскажите как с помощью функции serviceDocxGenerator, построить таблицу с позициями, которые будут заключены в разделы? Список позиций получается отобразить

**Comment by charados** [comment]
![WINWORD_RSSqv2YDGm](https://github.com/user-attachments/assets/af4a548d-ed5b-4975-8583-8ad79b218d25)

Как представлено на фотографии

**Reply by ttm-alexeypolunin** [reply, marked as answer]
Сгенерировать строки разделов в датасете

**Reply by charados** [reply]
А в worde переменные как задать? В верхнем поле переменную разделов, а в нижнем позиций как на фото сделал?

![browser_B0JYc3pcVC](https://github.com/user-attachments/assets/7659fb32-e3df-4b1c-9089-8f4432704681)

**Reply by ttm-alexeypolunin** [reply]
Так как на картинке — это только если у вас разделы фиксированные. Если они динамические, то только добавить в данные как название с пустыми прочими полями.

**Reply by charados** [reply]
А форматирование если отличается разделов и позиций?
